lua/cps.lua

```diff
@@ -15,6 +24,8 @@ return function (net_conf, gk_conf, gt_conf, lls_conf, numa_table)
 	-- Setup the CPS functional block.
 	cps_conf.lcore_id = gatekeeper.alloc_an_lcore(numa_table)
 	cps_conf.tcp_port_bgp = tcp_port_bgp
+	cps_conf.mailbox_max_entries = math.pow(2, mailbox_max_entries_exp)
```

Comment:
The field `mailbox_max_entries` should be named `mailbox_max_entries_exp` and receive the value of the variable `mailbox_max_entries_exp`. The function that initializes mailboxes should use `1 << mailbox_max_entries_exp` to create the ring, and `(1 << mailbox_max_entries_exp) - 1` to create the memory pool.